test/specs/reference/tests/18.comment_display_inline.js

```diff
@@ -0,0 +1,189 @@
+exports.execute = function (options) {
```

Comment:
Can you please add a comment before this line, to explain what are the different cases that you're testing? (for both of the specs that you've added)

You can summarize the tables that you have and what are you testing for each of the columns. I tried to do on the specs that are testing a lot of different scenarios, for example [this one](https://github.com/informatics-isi-edu/ermrestjs/blob/master/test/specs/column/tests/03.pseudo_column.js#L1). You don't need to explain everything, just some high-level details should be enough.